Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -198,9 +200,26 @@ namespace Fuse.Gestures
 		{
 			get { return _scrollable == null ? GesturePriority.Lower : _scrollable.GesturePriority; }
 		}
+
+		bool _pressed;
+		void StartInvalidateVisual()
+		{
+			if (!_pressed)
+				UpdateManager.AddAction(_scrollable.InvalidateVisual);
```

Comment:
CachingMode=Never might be too much. There are plenty of use-cases for ScrollView that don't involve the whole screen, or where the cotent remains static for a long-time. I think this stop-gap is probably better for now. 

If we do see this problem in other places we might consider attaching this to the interacction state of a control. Any control where `IsInteracting` is true would turn off caching.